### ISSUE TITLE
Remove GOREL relation 'results_in_division_of' ? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#18232]
The GOREL relation 'results_in_division_of' is not used anywhere in the ontology. The logical definition of children of cell division use 'occurs_in', or 'has_input' for example:

GO:0007282
name: cystoblast division
intersection_of: GO:0051301 ! cell division
intersection_of: **occurs_in** CL:0000722 ! cystoblast

id: GO:0048860
name: glioblast division
intersection_of: GO:0051301 ! cell division
intersection_of: **occurs_in** CL:0000030 ! glioblast

id: GO:0055057
name: neuroblast division
intersection_of: GO:0051301 ! cell division
intersection_of: **has_input** CL:0000031 ! neuroblast

id: GO:0017145
name: stem cell division
intersection_of: GO:0051301 ! cell division
intersection_of: **has_input** CL:0000034 ! stem cell

1. Which is the relation we should use ?
2. Do we need the relation 'results_in_division_of' ? 

@ukemi @cmungall 

Thanks, Pascale

**Comment by ukemi** [comment]
That's odd. These must have been changed at some time. has_input seems more correct than occurs_in.
@cmungall?

**Comment by vanaukenk** [comment]
Note that we have used 'results in division of' in annotation extensions.

**Comment by ukemi** [comment]
Yes. They are some of the ones that are still not passing the current specs.

**Comment by pgaudet** [comment]
From #18804 
see
http://wiki.geneontology.org/index.php/Ontology_meeting_2020-02-03#Aligning_Ontology.2C_Annotation.2C_ShEx

From 2020-01-27 call:
'results in division of' (GOREL:0001019)

    Not used in the ontology
        In the ontology, logical definitions use:
            'occurs in', e.g. cystoblast division = cell division and 'occurs in' some cystoblast
            'has input', e.g. germ-line stem cell division = cell division and 'has input' some germ-line stem cell
            loosely, but not exclusively, along somatic vs stem cell lines
        No design pattern (that I could find)

    For annotation, it looks like curators use 'occurs in', 'has participant, 'results in division of' although there aren't many annotation extensions at all for 'cell division' and children

    PROPOSED ACTION: review and update annotations to use 'has input' (create go-annotation ticket - couldn't find one already)
        Note that we will need to update the local range for 'has input' to allow for cells before doing this
    PROPOSED ACTION: obsolete 'results in division of' see https://github.com/geneontology/go-ontology/issues/18804
    PROPOSED ACTION: keep 'regulates o results in division of' for now
            fate of the chained relations is still undecided
            we have a model for importing them
            we haven't decided on exporting them, but are leaning towards not doing that
    PROPOSED ACTION: create a design pattern for cell division using 'has input'
